assets/scripts/vue/components/signalement-form/components/SignalementFormAddress.vue

```diff
@@ -189,6 +194,24 @@ export default defineComponent({
         }, 200)
       }
     },
+    handleKeyboardSuggestion (event: any, index: number) {
+      if (event.key === ' ' || event.key === 'Spacebar') {
+        this.handleClickSuggestion(index)
+      } else if (event.key === 'ArrowUp' || event.key === 'ArrowDown') {
+        event.preventDefault()
+        const suggestionElements = this.$refs.addressSuggestions as HTMLElement[]
+        if (suggestionElements && suggestionElements.length > 0) {
+          let newIndex = index
+          if (event.key === 'ArrowUp') {
+            newIndex = (index === 0) ? suggestionElements.length - 1 : index - 1
+          } else if (event.key === 'ArrowDown') {
+            newIndex = (index === suggestionElements.length - 1) ? 0 : index + 1
+          }
+          const nextElement = suggestionElements[newIndex] as HTMLElement
+          nextElement.focus()
+        }
+      }
+    },
```

Comment:
Pourquoi ne pas avoir récupérer ce qui a été fait pour `SignalementFormAutocomplete` qui utilise les directives vue pour gérer les événements ?
Et on devrait pouvoir s'en sortir sans l'usage de this.$refs ici